plugin/graphml/pom.xml

```diff
@@ -114,6 +114,17 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.graphstream</groupId>
+            <artifactId>gs-core</artifactId>
+            <version>1.3.1-SNAPSHOT</version>
```

Comment:
- a released version of the library should be used (i.e. no snapshot)
- if two dependencies of the same library need the same version this one should be declared as a property